service/callback_helper.go

```diff
@@ -0,0 +1,43 @@
+package service
+
+import (
+	"fmt"
+	"github.com/companieshouse/payments.api.ch.gov.uk/models"
+	"net/http"
+	"os"
+
+	"github.com/companieshouse/chs.go/log"
+)
+
+// redirectUser redirects user to the provided redirect_uri with query params
+func redirectUser(w http.ResponseWriter, r *http.Request, redirectURI string, state string, ref string, status string) {
```

Comment:
lot of string params required in this function call - would be nice to provide a simple struct to hold them all to ensure we don't end up with ordering issues for these params e.g.

```
type redirectParams struct {
    state string
    ref string
    status string
}
```

then the func definition becomes harder to mix up values as theres only 1 string input
```
func redirectUser(w http.ResponseWriter, r *http.Request, redirectURI string, params redirectParams) {
....
}
```